projects/packages/forms/src/blocks/field-select/edit.js

```diff
@@ -52,143 +47,48 @@ export default function DropdownFieldEdit( props ) {
 				'jetpack/input',
 				{ type: 'dropdown', placeholder: __( 'Select one option', 'jetpack-forms' ) },
 			],
+			[ 'jetpack/dropdown', { options, lock: { move: true, remove: true } } ],
 		];
-	}, [ required ] );
+	}, [ required, options ] );
 
 	const innerBlocksProps = useInnerBlocksProps(
 		{ className: 'jetpack-field-dropdown__wrapper' },
 		{
-			allowedBlocks: ALLOWED_INNER_BLOCKS,
+			allowedBlocks: [ ...ALLOWED_INNER_BLOCKS, 'jetpack/dropdown' ],
 			template,
 			templateLock: 'all',
 		}
 	);
-
+	/*
```

Comment:
I assume we can remove this commented out code.

Reply:
Thanks! Yep I'm still playing with code so sometimes leaving  things as commented, but I'll clean it up at the end.